src/protocols/bdx/BdxMessages.cpp

```diff
@@ -170,7 +169,7 @@ size_t TransferInit::MessageSize() const
 void TransferInit::LogMessage(bdx::MessageType messageType) const
 {
     char fd[kMaxFileDesignatorLen];
-    snprintf(fd, sizeof(fd), "%.*s", static_cast<int>(FileDesLength), FileDesignator);
```

Comment:
I think it was correct. "%.*s" requires `int`.

Reply:
I'll change it back but does it need to be? There is no compilation error nor did CI fail...

Reply:
I'm not sure, but I saw people fixing that in the past as some more strict compilers would throw a warning otherwise.